src/cephadm/cephadm

```diff
@@ -137,10 +138,12 @@ def call(command, desc=None, verbose=False, **kwargs):
 
     :param verbose_on_failure: On a non-zero exit status, it will forcefully set
                                logging ON for the terminal
+    :param timeout: timeout in seconds
     """
     if not desc:
         desc = command[0]
     verbose_on_failure = kwargs.pop('verbose_on_failure', True)
+    timeout = kwargs.pop('timeout', args.timeout)
```

Comment:
(I'd be +1 for making `timeout` a real parameter and then change this to

```suggestion
    timeout = timeout or args.timeout
```

Reply:
To be consistent perhaps `verbose_on_failure` should also be a real parameter too (and only use `kwargs` for `Popen`)?

Reply:
`timeout` and `verbose_on_failure` are now real parameters